### ISSUE TITLE
Cdn для скачивание модов.

### ISSUE THREAD

**Issue by TheXaver** [GravitLauncher/Launcher#69]
Добавить возможность скачивать файлы по ссылке.

**Comment by TheXaver** [comment]
По url*

**Comment by gravit0** [comment]
Что Вы имеете ввиду? Канала вашего сервера не хватает для закачки модов?

**Comment by TheXaver** [comment]
Нет, у некоторых людей коннекты плохие с сервером и было бы удобно скачивать по url, ведь cdn сервера(пример akamai) стоит практически в каждом городе. И у них будет локальное скачивание.

**Comment by TheXaver** [comment]
![image](https://user-images.githubusercontent.com/18555344/48635696-f60ca900-e9d9-11e8-830a-38dc41aeb1d4.png)

**Comment by Zaxar163** [comment]
На сегодняшний день это невозможно, так как лаунчер не на вебсокетах. Позже может и подумаем.
